game-app/map-data/src/main/java/org/triplea/map/game/notes/GameNotesMigrator.java

```diff
@@ -41,16 +41,15 @@ public void extractGameNotes() {
     // If we have a number of game notes files to create, run in a progress dialog.
 
     final Runnable migrationJob =
-        () -> {
-          xmlFilesMissingGameNotesFile.forEach(
-              xmlGameFile -> {
-                final String gameNotes = readGameNotesFromXml(xmlGameFile);
-                final String fileNameToWrite = GameNotes.createExpectedNotesFileName(xmlGameFile);
-                final Path fileToWrite = xmlGameFile.resolveSibling(fileNameToWrite);
-                log.info("Writing game notes file: " + fileToWrite.toFile().getAbsolutePath());
-                FileUtils.writeToFile(fileToWrite, gameNotes);
-              });
-        };
+        () ->
```

Comment:
nit, recommend to keep changes consistent within a PR. This is seemingly a different kind of cleanup being done here.

Reply:
Ah yes, wanted to move this to a different branch, but it got here by accident, my bad.